README.md

```diff
@@ -20,21 +20,21 @@
 
 ## What is lakeFS
 
-lakeFS is an open source tool that transforms your object storage into a Git-like repository. It enables you to manage your data lake the way you manage your code.
+lakeFS is an open-source tool that transforms your object storage into a Git-like repository. It enables you to manage your data lake how you manage your code.
```

Comment:
```suggestion
lakeFS is an open-source tool that transforms your object storage into a Git-like repository. It enables you to manage your data lake the way you manage your code.
```
The first change is great, the second is probably ungrammatical.  I don't think any standard reference will suggest using "how" in this manner.

Reply:
that's what Grammarly suggested🤷🏻‍♂️

Reply:
I understand that is what Grammarly suggested.  However I am unsure of the change.  I shall make this change if Grammarly have a reference for it being (more) correct.  

They might be looking at _usage_.  I found [this](https://english.stackexchange.com/questions/101086/is-the-way-how-wrong) with some n-gram statistics, but unfortunately talks about "is the way I" vs. "is how I"; dropping "is" to get "the way I" vs. "how I" allows many more meanings.

Reply:
Some data: here's a [usage chart for manage the way you vs. manage how you](https://books.google.com/ngrams/graph?content=manage+the+way+you%2Cmanage+how+you&year_start=1970&year_end=2005&corpus=5&smoothing=3).  Generally "how" is more common (_much_ more so in the 19th century), but "the way" has its periods (and is going down right now).  For British English "the way" is on a massive binge.  Mostly the data seem quite noisy.

Reply:
it's not about being more or less correct. it's about being concise and clear

Reply:
I understand and appreciate, really.  And I am happy to chalk it up to varying English dialects (American/British, probably).

Note that we are already [using this sentence externally and extensively](https://google.com/search?q="manage+your+data+the+way+you+manage+your+code").  I would like to rope in @einato on this.  But I would also like to merge everything else on the PR without waiting any longer.  Would it be OK to revert this change here, and I will open it in another PR?